plugins/source/aws/resources/services/s3/bucket_lifecycles.go

```diff
@@ -22,9 +22,10 @@ func bucketLifecycles() *schema.Table {
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			{
-				Name:     "bucket_arn",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: schema.ParentColumnResolver("arn"),
+				Name:       "bucket_arn",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   schema.ParentColumnResolver("arn"),
+				PrimaryKey: true,
```

Comment:
We should verify the API for Bucket lifecycle always returns a single item

Reply:
Added `ID` to PK in c2825823e5a4d53ed753f7cc26e2867e8e0ae0d1